### PR TITLE
Add htmleditor specific path

### PR DESCRIPTION
**Pull request by alex-deaconu**
Routing issue caused by updated Angular/ui-router versions

Fix by adding `/htmleditor` path
Update controller to force digest cycle on `$destroy`

[stage-2]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
New ui-router version 0.2.17.

@ezequielc please review. Thanks!

File: web/scripts/editor/controllers/ctr-html-editor.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ angular.module('risevision.editor.controllers')
       $scope.$on('$destroy', function () {
         presentationParser.parsePresentation(editorFactory.presentation);
         distributionParser.parseDistribution(editorFactory.presentation);
+        
+        $scope.$apply();
```
Comment (ezequielc) [inline web/scripts/editor/controllers/ctr-html-editor.js]: @alex-deaconu Just for curiosity, why do we need `$scope.$apply()` here? Thanks

Comment (ezequielc) [inline web/scripts/editor/controllers/ctr-html-editor.js]: @alex-deaconu Sorry, I missed that in your note.

**Comment by ezequielc** [comment]
@alex-deaconu :+1:  Thanks for fixing!

File: web/scripts/editor/controllers/ctr-html-editor.js (hunk 1 above)
Comment (alex-deaconu) [inline web/scripts/editor/controllers/ctr-html-editor.js]: @ezequielc no worries. I noticed that once the `$destroy` event was called, there was no `$digest` cycle to force updating the data for the Presentation. To be more specific, the test failure was this:
https://circleci.com/gh/Rise-Vision/rise-vision-apps/658

Nice catch from e2e tests!

Comment (ezequielc) [inline web/scripts/editor/controllers/ctr-html-editor.js]: @alex-deaconu Nice! Thanks!

**Pull request merged**